emcc.py

```diff
@@ -59,6 +59,13 @@
 
 logger = logging.getLogger('emcc')
 
+# In git checkouts of emscripten `bootstrap.py` exists to run post-checkout
+# steps.  In packaged versions (e.g. emsdk) this file does not exist
+# and these steps are assumed to have been run already.
```

Comment:
What makes `bootstrap.py` not exist there? Do those other places need to delete it manually?

Reply:
Its excluded by install.py (which also takes case of excluding a bunch of other stuff such as test data).

I added a comment.

I also added an extra check for `.git` just in case somebody is packaging without using `install.py`.